src/data/oppfolgingsplan/oppfolgingsplanQueryHooks.ts

```diff
@@ -70,3 +78,52 @@ export const useDokumentinfoQuery = (oppfolgingsplanId: number) => {
     staleTime: minutesToMillis(60 * 12),
   });
 };
+
```

Comment:
Vet ikke om vi ønsker å ha disse👇 i separat fil ettersom de gjelder for egen app 🤔

Reply:
Siden denne heter `oppfolgingsplanQueryHooks` så tenker jeg den skal inneholde hooks for å hente data om oppfølgingsplaner. Sånn sett tenker jeg de nye hookene hører hjemme i egne filer. 
Synes også det blir litt misvisende å ha query i navnet på hooks som endrer data (mutations) (ref `usePostOppfolgingsplanForesporselQuery`). I mitt hode er queries bare hooks som henter data. Så kanskje rename den til `usePostOppfolgingsplanForesporsel` ?

Reply:
Hva med renaming av filen til `oppfolgingsplanHooks`? Virker litt dumt å splitte GET og POST kallet til samme app 🤔 Eventuelt en egen fil for disse 2? Andre forslag?

Reply:
Kan godt ha en egen fil for disse to, noe med `oppfolgingsplanForesporselHooks` :+1: